src/test/java/org/example/WysiwygEditorIT.java

```diff
@@ -0,0 +1,43 @@
+package org.example;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class WysiwygEditorIT {
+
+    private static WebDriver driver;
+
+    @BeforeAll
+    static void setup() {
+
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+
+        driver.get("http://the-internet.herokuapp.com/tinymce");
+    }
+
+    @Test
+    void testingWysiwygEditorVerifiesThatContentIsAdded(){
+
+        // Act
+        String expectedContent = "Here is some content!";
+
```

Comment:
A tiny suggestion is to make the content something that was obviously written to be part of this test....

Maybe:  Some text for entered for Automated Testing